### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by kentmclean**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["env"],
+  "plugins": ["angularjs-annotate"]
```
Comment (kentmclean) [inline .babelrc:3]: this file added to bring in the angular-annotate plugin, which replaces the ng-annotate plugin, which replaces the ng-min plugin

File: package.json [hunk 2]
```diff
@@ -20,29 +19,29 @@
...
+    "webpack": "^3.10.0"
   },
   "dependencies": {
-    "angular": "^1.5.2",
-    "babel": "^6.5.2",
-    "jquery": "^2.2.2",
-    "lodash": "^4.6.1"
+    "angular": "^1.6.8",
+    "babel-cli": "^6.26.0",
+    "jquery": "^3.3.1",
```
Comment (kentmclean) [inline package.json:43]: THE whole reason to update this project. Updating jQuery to eliminate a known vulnerability in the older version

File: webpack.config.js [hunk 3]
```diff
@@ -1,41 +1,63 @@
 var path = require('path');
 var webpack = require('webpack');
-var ngminPlugin = require('ngmin-webpack-plugin');
 
 var version = require('./package.json').version;
 var name = require('./package.json').name;
 
 module.exports = {
-	entry: "./src/index.js",
```
Comment (kentmclean) [inline webpack.config.js:9]: upgraded webpack from 1.x to 3.x

**Comment by neiloler** [comment]
👍

**Pull request merged**